packages/controllers/src/snaps/RequestQueue.ts

```diff
@@ -0,0 +1,47 @@
+export class RequestQueue {
+  public readonly maxQueue: number;
+
+  private readonly locks: Map<string, number>;
```

Comment:
"locks" is a hangover from a previous implementation. `queueSizes` would be a better name.